src/averageAges.js

```diff
@@ -14,12 +14,23 @@
  *
  * @return {number}
  */
+const MALE_SIGN = 'm';
+const FEMALE_SIGN = 'f';
+
+function countAverageAge(arr) {
+  return arr.reduce((sum, { died, born }) =>
+    sum + (died - born), 0) / arr.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => {
+    return century
```

Comment:
We don't need `return` if function returns result without any calculations
`() => { return ... } === () => ...`
Same in all code